bleak/backends/bluezdbus/advertisement_monitor.py

```diff
@@ -20,11 +21,27 @@
 from dbus_fast import PropertyAccess
 from dbus_fast.service import ServiceInterface, dbus_property, method
 
-from bleak.args.bluez import OrPatternLike
+from bleak.args.bluez import OrPattern as _OrPattern
+from bleak.args.bluez import OrPatternLike as _OrPatternLike
 from bleak.backends.bluezdbus import defs
 
 logger = logging.getLogger(__name__)
 
+_DEPRECATED: dict[str, Any] = {
+    "OrPattern": _OrPattern,
+    "OrPatternLike": _OrPatternLike,
+}
+
+
+def __getattr__(name: str):
+    if value := _DEPRECATED.get(name):
+        warn(
+            f"importing {name} from bleak.backends.bluezdbus.advertisement_monitor is deprecated",
+            DeprecationWarning,
+        )
```

Comment:
Please check the stacklevel for these. I believe it will need to be `2` or even `3`. Otherwise the warning won't be that helpful if you don't know where it's coming from.